saleor/plugins/webhook/plugin.py

```diff
@@ -1077,7 +1133,12 @@ def _trigger_promotion_rule_event(
                 }
             )
             trigger_webhooks_async(
```

Comment:
Just a hint, 
You could create the webhook manager method like:
```
def trigger_webhooks_async(self, *args, **kwargs):
    return trigger_webhooks_async(*args, **kwargs, allow_replica=self.allow_replica)
```
Then replace in the code `trigger_webhooks_async` to `self. trigger_webhooks_async)`